prime-router/metadata/schemas/PrimeDataInput/pdi-covid-19.schema

```diff
@@ -17,12 +17,12 @@ elements:
     csvFields: [{name: Patient_suffix}]
 
   # Patient ID is generated internally by SimpleReport
-  - name: simple_report_patient_id
+  - name: standard.patient_id
     type: TEXT
     csvFields: [{name: Patient_ID}]
 
   # In SimpleReport, lookup_id is the ID the user entered into the app.
-  - name: standard.patient_id
+  - name: patient_lookup_id
```

Comment:
SimpleReport is not supplying the patient_lookup_id right now, so I'm rolling back this change until we can clarify with SimpleReport team, and coordinate on timing.